spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkV2Filters.java

```diff
@@ -302,10 +326,25 @@ private static <T> T childAtIndex(Predicate predicate, int index) {
     return (T) predicate.children()[index];
   }
 
+  private static boolean couldConvert(org.apache.spark.sql.connector.expressions.Expression expr) {
```

Comment:
How about using `canConvertToTerm` instead? That fixes a couple of issues with the name `couldConvert`:
1. Literals can be converted, but this doesn't allow literals.
2. The word "could" is past tense, implying that the method actually converts and then returns whether conversion was successful. But this is just checking if the conversion would succeed.

Reply:
changed to `canConvertToTerm `